engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions.controller.js

```diff
@@ -39,11 +39,6 @@ angular.module('Bastion.subscriptions').controller('SubscriptionsController',
             $scope.transitionTo('subscriptions.index');
         };
 
-        $scope.groupedSubscriptions = {};
-        $scope.$watch('table.rows', function (rows) {
-            $scope.groupedSubscriptions = SubscriptionsHelper.groupByProductName(rows);
-        });
-
```

Comment:
The page looks the same as before, I just think there is a simpler way to achieve it. 

![](http://i.imgur.com/ohHx8Bl.png)
![](http://i.imgur.com/3G7wHso.png)